src/Discord.Net.Core/Entities/Interactions/MessageComponents/ComponentType.cs

```diff
@@ -26,8 +26,23 @@ public enum ComponentType
         TextInput = 4,
 
         /// <summary>
-        ///     An interaction sent when a model is submitted.
+        ///     A select menu for picking from users.
         /// </summary>
-        ModalSubmit = 5,
+        UserSelect = 5,
+
+        /// <summary>
+        ///     A select menu for picking from roles.
+        /// </summary>
+        RoleSelect = 6,
+
+        /// <summary>
+        ///     A select menu for picking from roles and users.
```

Comment:
Does this not also include channels? Or am I mistaken?
```suggestion
        ///     A select menu for picking from roles, channels, and users.
```
> In case of being pointed to the extra added comma, I've been told its in accordance with the oxford comma rule, lol

Reply:
Mentionable select type only includes entities you can **mention** - roles & users